recipe/meta.yaml

```diff
@@ -33,22 +31,22 @@ requirements:
     - python
     - pip
     - setuptools
-    - llvmlite 0.31.*
+    - llvmlite 0.31.0
     - numpy
-    - tbb-devel >=2018.0.5  # [linux]
+    - tbb-devel >=2019.5  # [linux]
 
   run:
     - python
-    - {{ pin_compatible('llvmlite', max_pin='x.x') }}
-    - {{ pin_compatible('numpy') }}
+    - llvmlite >=0.31.0,<0.33
+    - {{ pin_compatible('numpy', upper_bound='1.18.0') }}
```

Comment:
@jjhelmus, does this look ok to you? Guessing we will need to hot-fix previous `numba` packages to have this constraint as well.

Reply:
How comes there's no such limitation in https://github.com/numba/numba/blob/master/setup.py?

Reply:
I don't know. FWIW we raised issue ( https://github.com/numba/numba/issues/5752 ), which it looks like @stuartarchibald has already picked up. We are hoping this will be ironed out in the next release of Numba.

Reply:
Lack of NumPy 1.18 support comprises:
* 2 things that refuse to compile
* 1 thing which produces a 1.17 result to do with NaT changes 
* 1 really weird corner case with `argwhere(False)`

Am hoping to resolve this for 0.50.